hooks/webhook/webhook.go

```diff
@@ -77,8 +80,18 @@ func (i *Invoker) Invoke(request, response interface{}) error {
 		glog.Infof("%s: Will invoke %q", i, reqBodyIndent)
 	}
 
-	// Send request.
 	client := &http.Client{Timeout: i.Timeout}
+	if i.CABundle != "" {
```

Comment:
This maybe the right place, where we explain how this CABundle will be made available to the binary.
We should also add PR message that explain the same.